docs/sources/project/advanced-contributing.md

```diff
@@ -58,69 +71,69 @@ The following provides greater detail on the process:
-	`daemon/` folder. 
+    For example, if you want to redesign our daemon create a new file under the
+    `daemon/` folder. 
+
+6. Name the file descriptively, for example `redesign-daemon-proposal.md`.
 
-6. Name the file descriptively, for example `redesign-daemon-proposal.md`.	
-	
 7. Write a proposal for your change into the file.
 
-	This is a markdown file that describes your idea. Your proposal
-	should include information like:
-	
-	* Why is this changed needed or what are the use cases?
-	* What are the requirements this change should meet?
-	* What are some ways to design/implement this feature?
-	* Which design/implementation do you think is best and why?
-	* What are the risks or limitations of your proposal?
-	
-	 This is your chance to convince people your idea is sound. 
-	  
+    This is a Markdown file that describes your idea. Your proposal
+    should include information like:
+
+    * Why is this changed needed or what are the use cases?
+    * What are the requirements this change should meet?
+    * What are some ways to design/implement this feature?
+    * Which design/implementation do you think is best and why?
+    * What are the risks or limitations of your proposal?
+
+     This is your chance to convince people your idea is sound. 
+  
 8. Submit your proposal in a pull request to `docker/docker`.
 
-	The title should have the format:
-	
-	**Proposal:** _short title_
-	
-	The body of the pull request should include a brief summary of your change
-	and then say something like "_See the file for a complete description_".
-	
+    The title should have the format:
+
+    **Proposal:** _short title_
+
+    The body of the pull request should include a brief summary of your change
+    and then say something like "_See the file for a complete description_".
+
 9. Refine your proposal through review.
 
-	The maintainers and the community review your proposal. You'll need to
-	answer questions and sometimes explain or defend your approach. This is
-	chance for everyone to both teach and learn.
-   
+    The maintainers and the community review your proposal. You'll need to
+    answer questions and sometimes explain or defend your approach. This is
+    chance for everyone to both teach and learn.
+
 10. Pull request accepted.
 
-	Your request may also be rejected. Not every idea is a good fit for Docker.
-	Let's assume though your proposal succeeded. 
-	
+    Your request may also be rejected. Not every idea is a good fit for Docker.
+    Let's assume though your proposal succeeded. 
+
 11. Implement your idea.
 
-	Implementation uses all the standard practices of any contribution.
-	
-	 * fork `docker/docker`
-	 * create a feature branch
-	 * sync frequently back to master
-	 * test as you go and full test before a PR
-	 
-	 If you run into issues, the community is there to help.
-	 
+    Implementation uses all the standard practices of any contribution.
+
+     * fork `docker/docker`
```

Comment:
This list has the same problem as the last; blank line introduced and the terminating sentence is part of the numbered list and should not be indented this far.

![image](https://cloud.githubusercontent.com/assets/1347209/6541694/ff274b68-c493-11e4-98b9-db4ca120adf3.png)